actors/builtin/miner/miner_test.go

```diff
@@ -3020,7 +3020,7 @@ func TestExtendSectorExpiration(t *testing.T) {
 		// and prove it once to activate it.
 		advanceAndSubmitPoSts(rt, actor, sector)
 
-		maxLifetime, err := builtin.SealProofSectorMaximumLifetime(sector.SealProof)
+		maxLifetime, err := builtin.SealProofSectorMaximumLifetime(sector.SealProof, network.Version0)
```

Comment:
It should not matter in this case but typically unit tests adjust to cover the latest network version so best practice is to replace `network.Version0` with `network.VersionMax`